packages/aws-cdk/toolkit/lib/cli-io-host.ts

```diff
@@ -0,0 +1,143 @@
+import * as chalk from 'chalk';
+
+/**
+ * Basic message structure for toolkit notifications.
+ * Messages are emitted by the toolkit and handled by the IoHost.
+ */
+interface IoMessage {
+  /**
+   * @param time The time the message was emitted.
+   */
+  readonly time: Date;
+
+  /**
+   * @param level The log level of the message.
+   */
+  readonly level: IoMessageLevel;
+
+  /**
+   * @param action The action that triggered the message.
+   */
+  readonly action: IoAction;
+
+  /**
+   * @param code A short code to identify the message.
+   */
+  readonly code: string;
+
+  /**
+   * @param message The message text.
+   */
+  readonly message: string;
+
+  /**
+   * @param forceStdout If true, the message will be written to stdout regardless of log level.
```

Comment:
also regardless of ci or not. i think we should say regardless of any other parameters. if `forceStdout` is set, the message will be sent to stdout _no matter what_